Formula/sshping.rb

```diff
@@ -0,0 +1,24 @@
+class Sshping < Formula
+  desc "SSH based ping: measure character echo latency and bandwidth"
+  homepage "https://github.com/spook/sshping"
+  url "https://github.com/spook/sshping/archive/refs/tags/v0.1.4.tar.gz"
+  sha256 "589623e3fbe88dc1d423829e821f9d57f09aef0d9a2f04b7740b50909217863a"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pod2man" => :build
+  depends_on "libssh"
+
+  def install
+    rm "Makefile"
+    system "cmake", *std_cmake_args
+    system "make"
+    system "make", "install"
+    man8.install "doc/sshping.8"
```

Comment:
Just use a dedicated build directory (shouldn't need to worry about any other Makefile), let `cmake` handle everything, and move the installed manpages:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
    share.install prefix/"man"
```